core/src/main/resources/jenkins/split-plugins.txt

```diff
@@ -34,3 +34,6 @@ trilead-api 2.184 1.0.4
 
 # JENKINS-64107
 sshd 2.281 3.0.1
+
+javax-activation-api 2.328 1.2.0-2
+javax-mail-api 2.328 1.6.2-5
```

Comment:
https://groups.google.com/g/jenkinsci-advisories/c/wFaEbxbFSY0/m/KMza8OrfEAAJ + https://www.jenkins.io/security/scheduling/#changes-in-security-updates means that 2.329 will be released without this change, even if merged today.

```suggestion
javax-activation-api 2.329 1.2.0-2
javax-mail-api 2.329 1.6.2-5
```